### PR TITLE
Inform user about deleting consumers before deleting event types

### PR DESCRIPTION
**Pull request by thomasabraham**
Inform the users that they should delete the subscriptions before deleting event types.
This is because Nakadi will refuse to delete event types with active subscriptions.

### CODE REVIEW

**Comment by Kunal-Jha** [comment]
@thomasabraham  It looks good.  But don't you think we should also have some sort of warning for Nakadi SQL queries?

File: client/Pages/EventTypeDetails/View.elm [hunk 1]
```diff
@@ -821,7 +821,8 @@ deletePopup model eventType consumersStore subscriptionsStore appsInfoUrl usersI
                                 Nothing ->
                                     none
                             , p [ class "dc-p" ]
-                                [ text "Think twice, notify all consumers and producers."
+                                [ text "Notify all consumers and producers."
+                                , text "Delete all active subscriptions except the one for nakadi-archiver."
```
Comment (rcillo) [inline client/Pages/EventTypeDetails/View.elm]: @thomasabraham you might want to move this text to an environment variable as this details is not pertinent to all users of the open source Nakadi UI project. It's also not true for both staging and live environments.

**Comment by SergKam** [comment]
If you need any help feel free to ask me. I know that the testing set up a bit tricky here.

**Comment by rcillo** [comment]
@thomasabraham the code looks good. Could you please add a screenshot of the dialog, just to give an idea of where the new panel will be?

**Comment by rcillo** [comment]
👍

**Comment by thomasabraham** [comment]
👍

**Pull request merged**